attestation-ca/src/lib.rs

```diff
@@ -2,15 +2,31 @@ use base64urlsafedata::Base64UrlSafeData;
 use openssl::error::ErrorStack as OpenSSLErrorStack;
 use openssl::{hash, x509};
 use serde::{Deserialize, Serialize};
-use std::collections::{BTreeMap, BTreeSet};
+use std::collections::BTreeMap;
 
 use uuid::Uuid;
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct DeviceDescription {
+    pub(crate) en: String,
+    pub(crate) localised: BTreeMap<String, String>,
+}
+
+impl DeviceDescription {
+    pub fn description_en(&self) -> &str {
+        self.en.as_str()
+    }
+
+    pub fn description_localised(&self) -> &BTreeMap<String, String> {
```

Comment:
Describe the keys of this map here. I think they're derived verbatim from the MDS (`alternative_descriptions`), and they're never used in test data.